<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="photo_map">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>599</width>
      <height>449</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="search_line">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Вид карты</string>
     </property>
     <addaction name="to_map_act"/>
     <addaction name="to_sat_act"/>
     <addaction name="to_satskl_act"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="to_search_act"/>
    <addaction name="to_clear_s_act"/>
    <addaction name="separator"/>
    <addaction name="to_postcode_act"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>Схема</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Спутник</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Гибрид</string>
   </property>
  </action>
  <action name="to_search_act">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="to_clear_s_act">
   <property name="text">
    <string>Сброс поискового результата</string>
   </property>
  </action>
  <action name="to_map_act">
   <property name="text">
    <string>Схема</string>
   </property>
  </action>
  <action name="to_sat_act">
   <property name="text">
    <string>Спутник</string>
   </property>
  </action>
  <action name="to_satskl_act">
   <property name="text">
    <string>Гибрид</string>
   </property>
  </action>
  <action name="to_postcode_act">
   <property name="text">
    <string>Приписывание почтового индекса</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
